configuration/config-txt/video.md

```diff
@@ -158,168 +158,218 @@ The `hdmi_group` command defines the HDMI output group to be either CEA (Consume
+| 23 | 1280x768 | 60Hz | 15:9 | |
+| 24 | 1280x768 | 75Hz | 15:9 | |
+| 25 | 1280x768 | 85Hz | 15:9 | |
+| 26 | 1280x768 | 120Hz | 15:9 | reduced blanking |
+| 27 | 1280x800 | 60 | 16:10 | reduced blanking |
+| 28 | 1280x800 | 60Hz | 16:10 | |
+| 29 | 1280x800 | 75Hz | 16:10 | |
+| 30 | 1280x800 | 85Hz | 16:10 | |
+| 31 | 1280x800 | 120Hz | 16:10 |reduced blanking |
+| 32 | 1280x960 | 60Hz | 4:3 | |
+| 33 | 1280x960 | 85Hz | 4:3  | |
+| 34 | 1280x960 | 120Hz | 4:3  |reduced blanking |
+| 35 | 1280x1024 | 60Hz | 5:4 | |
+| 36 | 1280x1024 | 75Hz | 5:4 | |
+| 37 | 1280x1024 | 85Hz | 5:4 | |
+| 38 | 1280x1024 | 120Hz | 5:4 | reduced blanking |
+| 39 | 1360x768 | 60Hz | 16:9 | |
+| 40 | 1360x768 | 120Hz | 16:9 | reduced blanking |
+| 41 | 1400x1050 | 60Hz| 4:3 | reduced blanking |
+| 42 | 1400x1050 | 60Hz | 4:3 | |
+| 43 | 1400x1050 | 75Hz | 4:3 | |
+| 44 | 1400x1050 | 85Hz | 4:3 | |
+| 45 | 1400x1050 | 120Hz | 4:3 | reduced blanking |
+| 46 | 1440x900 | 60Hz | 16:10 | reduced blanking |
+| 47 | 1440x900 | 60Hz | 16:10 | |
+| 48 | 1440x900 | 75Hz | 16:10 | |
+| 49 | 1440x900 | 85Hz | 16:10 | |
+| 50 | 1440x900 | 120Hz | 16:10 |reduced blanking |
+| 51 | 1600x1200 | 60Hz | 4:3 | |
+| 52 | 1600x1200 | 65Hz | 4:3 | |
+| 53 | 1600x1200 | 70Hz | 4:3 | |
+| 54 | 1600x1200 | 75Hz | 4:3 | |
+| 55 | 1600x1200 | 85Hz | 4:3 | |
+| 56 | 1600x1200 | 120Hz | 4:3  | reduced blanking |
+| 57 | 1680x1050 | 60Hz | 16:10 | reduced blanking |
+| 58 | 1680x1050 | 60Hz | 16:10 | |
+| 59 | 1680x1050 | 75Hz | 16:10 | |
+| 60 | 1680x1050 | 85Hz | 16:10 | |
+| 61 | 1680x1050 | 120Hz | 16:10 | reduced blanking |
+| 62 | 1792x1344 | 60Hz | 4:3 |  |
+| 63 | 1792x1344 | 75Hz | 4:3 | |
+| 64 | 1792x1344 | 120Hz | 4:3 | reduced blanking |
+| 65 | 1856x1392 | 60Hz | 4:3 | |
+| 66 | 1856x1392 | 75Hz | 4:3 | |
+| 67 | 1856x1392 | 120Hz | 4:3 | reduced blanking |
+| 68 | 1920x1200 | 60Hz | 16:10  |reduced blanking |
+| 69 | 1920x1200 | 60Hz | 16:10  | |
+| 70 | 1920x1200 | 75Hz | 16:10  | |
+| 71 | 1920x1200 | 85Hz | 16:10  | |
+| 72 | 1920x1200 | 120Hz | 16:10  |reduced blanking |
+| 73 | 1920x1440 | 60Hz | 4:3 | |
+| 74 | 1920x1440 | 75Hz | 4:3 | |
+| 75 | 1920x1440 | 120Hz | 4:3 | reduced blanking |
+| 76 | 2560x1600 | 60Hz| 16:10 | reduced blanking |
+| 77 | 2560x1600 | 60Hz | 16:10 | |
+| 78 | 2560x1600 | 75Hz | 16:10 | |
+| 79 | 2560x1600 | 85Hz | 16:10 | |
+| 80 | 2560x1600 | 120Hz | 16:10 | reduced blanking |
+| 81 | 1366x768 | 60Hz | 16:9 |  |
+| 82 | 1920x1080 | 60Hz | 16:9 | 1080p |
+| 83 | 1600x900 | 60Hz | 16:9 | reduced blanking |
+| 84 | 2048x1152 | 60Hz | 16:9 | reduced blanking |
+| 85 | 1280x720 | 60Hz | 16:9 | 720p |
+| 86 | 1366x768 | 60Hz | 16:9 | reduced blanking |
+
+Note that there is a [pixel clock limit](https://www.raspberrypi.org/forums/viewtopic.php?f=26&t=20155&p=195443#p195443).The highest supported mode on models prior to the Raspberry Pi 4 is 1920x1200 at 60Hz with reduced blanking, whilst the Raspberry Pi 4 can support up to 4096x2160 at 60Hz.
```

Comment:
> whilst the Raspberry Pi 4 can support up to 4096x2160 at 60Hz.

IIRC that's if only a single display-output is being used? Is it worth mentioning the maximum resolution/frequency if both HDMI outputs are being used?

Reply:
Good idea. Will do.